RELEASE_NOTES.txt

```diff
@@ -4,4 +4,43 @@
 -------------------------------------------------------------------------------------------
 ======================================================
 
-Work in progress.
\ No newline at end of file
+==========================================
+New Features
+==========================================
+
+- Added new package Zeiss.PiWeb.Api.Core
+○ contains highly used core objects with own logic which are no longer simple DTOs, e.g. Attribute, PathInformation, PathElement, InspectionPlanEntity enum as well as their extensions and PathHelper class
```

Comment:
I recommend to write "commonly used core objects" to indicated they are supposed to be used from different code places.

Reply:
Done